src/client/dfuse/dfuse_core.c

```diff
@@ -432,6 +434,12 @@ dfuse_destroy_fuse(struct dfuse_projection_info *fs_handle)
 			       refs, handles);
 	}
 
+	rc = daos_eq_destroy(fs_handle->dpi_eq, DAOS_EQ_DESTROY_FORCE);
```

Comment:
I/O should be acquiesced at this point so force should not be necessary, it might be better to not provide this which would presumably mean if there were any outstanding I/Os then they'd raise an error rather than being silently removed.

Reply:
ok will update